assets/js/modules/analytics/settings/settings-edit.js

```diff
@@ -45,29 +45,8 @@ export default function SettingsEdit() {
 	const canSubmitChanges = useSelect( ( select ) => select( STORE_NAME ).canSubmitChanges() );
 	const isDoingGetAccounts = useSelect( ( select ) => select( STORE_NAME ).isDoingGetAccounts() );
 	const isDoingSubmitChanges = useSelect( ( select ) => select( STORE_NAME ).isDoingSubmitChanges() );
-	const haveSettingsChanged = useSelect( ( select ) => select( STORE_NAME ).haveSettingsChanged() );
 	const isCreateAccount = ACCOUNT_CREATE === accountID;
 
-	// Rollback any temporary selections to saved values on dismount.
-	// This is a bit of a hacky solution, as we'd prefer to rollback changes
-	// when the "cancel" button is clicked. But we don't yet have control over
-	// that section of the page, so if the component is unmounted, has changes,
-	// and is not submitting those changes: we rollback.
-	//
-	// Technically this means we rollback right before we receive new settings
-	// when they ARE saved, because the component is unmounted then and meets the
-	// `haveSettingsChanged && ! isDoingSubmitChanges` criteria below.
-	// But that's fine as the new settings are then immediately loaded into state
-	// and there aren't any visual glitches. 🤷🏻‍♂️
-	const { rollbackSettings } = useDispatch( STORE_NAME );
-	useEffect( () => {
-		return () => {
-			if ( haveSettingsChanged && ! isDoingSubmitChanges ) {
```

Comment:
Just a question, how did this not work before? Wouldn't `isDoingSubmitChanges` be true while creating property or profile is still happening?